lib/common/client/errors.rb

```diff
@@ -14,6 +14,10 @@ def initialize(message = nil, status = nil, body = nil)
           @status = status
           @body = body
         end
+
+        def deconstruct
+          [message, status, body]
+        end
```

Comment:
✂️ You can remove this line since we decided against pattern matching the response.

Reply:
This is accurate, no need for this